fda-model-test/integrationtests/BootstrapToPairedDataShould.cs

```diff
@@ -0,0 +1,140 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using alternatives;
+using compute;
+using interfaces;
+using metrics;
+using paireddata;
+using scenarios;
+using Statistics;
+using Statistics.Distributions;
+using Statistics.Histograms;
+using Xunit;
+using System.Diagnostics;
+
+namespace fda_model_test.integrationtests
+{
+    public class BootstrapToPairedDataShould
+    {
+
+        //I am including this method here because I want to be able to compare flow dists between 1.4.3 and 2.0 
+        //without all of the simulation stuff getting in the way 
+        public IPairedData BootstrapToPairedData(IProvideRandomNumbers randomProvider, ContinuousDistribution continuousDistribution, int ordinates)
+        {
+            double[] samples = randomProvider.NextRandomSequence(continuousDistribution.SampleSize);
+            IDistribution bootstrap = continuousDistribution.Sample(samples);
+            //for (int i = 0; i < dist.SampleSize; i++) samples[i] = Math.Log10(dist.InverseCDF(samples[i]));
+            //ISampleStatistics ss = new SampleStatistics(samples);
+            double[] x = new double[ordinates];
+            double[] y = new double[ordinates];
+            //double skewdividedbysix = ss.Skewness / 6.0;
```

Comment:
Dead code, but this is a unit test, so not that important.

Reply:
fixed